config/locales/fr.yml

```diff
@@ -0,0 +1,175 @@
+fr:
+  title: "RubyGems.org"
+  subtitle: "votre communauté d'hébergement des gems"
+  help_url: "http://help.rubygems.org"
+  please_sign_up: "Accès refusé. Inscrivez-vous sur https://rubygems.org"
+  feed_subscribed: "RubyGems.org | Gems abonnés"
+  feed_latest: "RubyGems.org | Derniers Gems"
+  time_ago: "il y a %{duration}"
+  form_disable_with: "Veuillez patienter..."
+  update: "Mise à jour"
+  edit: "Modification"
+  sign_in: "Connexion"
+  sign_up: "Inscription"
+  footer_about: "RubyGems.org est le service d&rsquo;hébergement de la communauté Ruby. Publiez vos gems instantanément et utilisez-les. Utilisez l'API pour interagir et trouver des informations sur les gems disponibles. Contribuez et améliorez ce site avec nous !"
+  locale_name: "Français"
+  this_rubygem_could_not_be_found: "Ce rubygem n'a pas pu être trouvé."
+
+  dashboards:
+    show:
+      latest: "Dernières mises à jour"
+      latest_title: "Flux RSS des dernières mises à jour"
+      no_subscriptions: "Vous ne vous êtes abonné à aucun email. Visitez %{gem_link} pour vous y abonner !"
+      gem_link_text: "la page d'un gem"
+      mine: "Mes Gems"
+      no_owned: "Vous n'avez pas encore publié de gem. Vous pouvez vérifier les guides de %{creating_link} ou de %{migrating_link} de gems depuis RubyForge."
+      creating_link_text: "création"
+      migrating_link_text: "migration"
+      my_subscriptions: "Abonnements"
+
+  home:
+    index:
+      downloads_counting: "téléchargements &amp; comptages"
```

Comment:
![image](https://cloud.githubusercontent.com/assets/3535/11380783/27949438-92c6-11e5-8541-78f6d6833f19.png)

That's the context, it could be "Déjà %{n} téléchargements" with some code changes or simply "téléchargements à ce jour"